src/custom/hooks/useGetReceipt.ts

```diff
@@ -0,0 +1,45 @@
+import { useCallback } from 'react'
+import { useActiveWeb3React } from 'hooks/web3'
+import { retry, RetryableError, RetryOptions } from 'utils/retry'
+import { TransactionReceipt } from '@ethersproject/abstract-provider'
+
+const DEFAULT_RETRY_OPTIONS: RetryOptions = { n: 3, minWait: 1000, maxWait: 3000 }
+const RETRY_OPTIONS_BY_CHAIN_ID: { [chainId: number]: RetryOptions } = {
```

Comment:
this is an empty object

Reply:
Yep, i know, we don't have any override of the retry logic. I left as comment some examples from the original code